redisvl/extensions/session_manager/schema.py

```diff
@@ -24,6 +24,8 @@ class ChatMessage(BaseModel):
     """An optional identifier for a tool call associated with the message."""
     vector_field: Optional[List[float]] = Field(default=None)
     """The vector representation of the message content."""
+    dtype: Optional[str] = Field(default="float32")
```

Comment:
Same note as the other pydantic model about having this passed in by the extension class when needed